src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/CollectionDataContract.cs

```diff
@@ -68,14 +68,16 @@ public T Value
 
     [DataContract(Namespace = "http://schemas.microsoft.com/2003/10/Serialization/Arrays")]
 #if USE_REFEMIT
-    public struct KeyValue<K, V>
+    public class KeyValue<K, V>
 #else
```

Comment:
Agree with @Zhenlan. I'm curious about the perf difference between desktop and .NET core after this change were made on both. There may be some hotspots not yet identified. LGTM.